main.cpp

```diff
@@ -1,12 +1,35 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
```

Comment:
2018 

add SPDX identifier. it shall be enough with just SPDX identifier and copyright line